lib/Drush/Boot/DrupalBoot8.php

```diff
@@ -43,7 +48,9 @@ function bootstrap_drupal_database() {
   }
 
   function bootstrap_drupal_configuration() {
-    $this->drupal8_bootstrap(DRUSH_BOOTSTRAP_DRUPAL_CONFIGURATION);
+    $request = Request::createFromGlobals();
+    $classloader = drush_drupal_load_autoloader(DRUPAL_ROOT);
```

Comment:
Honestly, this looks wrong, because this is the bootstrap for a regular request.